core/jazz_admin/index.js

```diff
@@ -0,0 +1,109 @@
+// =========================================================================
+// Copyright ©  2017 T-Mobile USA, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// =========================================================================
+
+/**
+API for the admin user to retrieve the installer vars JSON
+@author:
+@version: 1.0
+ **/
+
+'use strict';
+const errorHandlerModule = require("./components/error-handler.js"); //Import the error codes module.
+const responseObj = require("./components/response.js"); //Import the response module.
+const configObj = require("./components/config.js"); //Import the environment data.
+const logger = require("./components/logger.js"); //Import the logging module.
+const request = require('request');
+function handler(event, context, cb) {
+  //Initializations
+  var errorHandler = errorHandlerModule();
+  var config = configObj(event, context);
+  logger.init(event, context);
+
+  try {
+    apiResponseObj = {};
+    if (event && event.method && event.method === 'GET') {
+      if (!event.principalId) {
+        logger.error('Authorizer did not send the user information, please check if authorizer is enabled and is functioning as expected!');
+        return cb(JSON.stringify(errorHandler.throwUnauthorizedError("User is not authorized to access this service|Authorization Incomplete")));
+      }
+      if (event.principalId != config.ADMIN_ID) {
+        return cb(JSON.stringify(errorHandler.throwUnauthorizedError("This User does not have the privileges to  access this service")));
+      }
+      getInstallerVarsJSON(config).then((data) => {
```

Comment:
We need input validation here to see if the user is requesting GET on `/admin/config`

Reply:
What is the validation that is required? -
the path is changed in the swagger file as {service-name}/config - and is the only path
it would only be directed to this API only if it is coming under admin/config